src/shared/components/Search/Search.tsx

```diff
@@ -0,0 +1,171 @@
+import styled from '@emotion/styled'
```

Comment:
I just rebased `discovery` with `dev`. Recently we've added some additional styles to the search component so this would need to be reflected here. Although I'm a bit unsure whether we should stick that into `Search` component. Design seems to define specific visuals just for the Channel search component. Maybe for now we should keep it in the channel view since we won't be using it here anyway?